docs/book/src/developer/tilt.md

```diff
@@ -230,15 +231,15 @@ Example:
 
 ```yaml
 extra_args:
-  core: ["--feature-gates=MachinePool=true"]
-  kubeadm-bootstrap: ["--feature-gates=MachinePool=true"]
-  azure: ["--feature-gates=MachinePool=true"]
+  core: ["--logging-format=json"]
```

Comment:
feature gates are ideally enabled via kustomize_substitutations (we have an example above). So I just used logging-format as an example here

Reply:
nit:
for sake of simplicity what about having only one manager as an example and expanding the array on a new line (dropping the [] notation)

Reply:
Done